recipes-bsp/device-tree/device-tree.bbappend

```diff
@@ -19,6 +19,14 @@ SRC_URI_append_uz7ev-evcc-hdmi = "\
 	file://hdmi.dtsi \
 "
 
+SRC_URI_append_uz7ev-evcc-hdmi-v = "\
```

Comment:
not needed as uz7ev-evcc-hdmi-v is a uz7ev-evcc-hdmi machine, so it already has the hdmi.dtsi

Reply:
OK, removed